util/config_stripper.py

```diff
@@ -66,8 +81,10 @@ def strip_tar(input, output):
     files_to_add = []
     for root, _, files in os.walk(tempdir):
         for f in files:
-            name = os.path.join(root, f)
-            files_to_add.append(name)
+            if os.path.basename(f) != 'repositories':
```

Comment:
Maybe a "whitelist" would be more clear here? We only want to do this to .tar.gz files right?